packages/create-block/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Enhancements
+
+- Update `esnext` (default) template to integrate with CSS import support added to `@wordpress/scripts`.
```

Comment:
I would change the wording here to something like: 
```suggestion
- Update `esnext` (default) template to leverage CSS import support added to `@wordpress/scripts`.
```

Reply:
Updated 👍